docs/guides/nxp_k32w_android_commissioning.md

```diff
@@ -0,0 +1,451 @@
+
+CHIPTool is now ready to be used for commissioning.
+
+<hr>
+
+<a name="form-thread"></a>
+
+## Forming a Thread network on the Border Router
+
+1.  On the mobile phone connect to the _OT-BR_ Wi-Fi network.
+
+2.  Open a browser and type the IP address 192.168.4.1. The Thread Border Router
+    Web Management page will open.
+
+3.  Navigate to the _Form_ tab then push the _Form_ button using the default
+    parameters:
+    ![nxp_form_nwk](../../examples/platform/k32w/doc/images/form_web.JPG)
+
+4.  A message "Form operation is successful" should be display after a few
+    seconds.
+
+5.  In case you ever want to reset these parameters issue this command from the
+    border router shell:
+
+        $ sudo ot-ctl factoryreset
+
+6.  Info: For debugging the Border Router, _ot-ctl_ offers an entry point to
+    [Thread CLI Commands](https://github.com/openthread/openthread/blob/master/src/cli/README.md).
+
+7.  In case there is any issue with the Web GUI check the logs on the Border
+    Router side / restart the daemon:
+
+        $ sudo service otbr-agent status
+        $ sudo service otbr-agent restart
+
+## Preparing accessory device
+
+To prepare the accessory device for commissioning, complete the following steps:
+
+1.  Make sure that JP4 and JP7 jumpers are in leftmost position and a mini-USB
+    cable is connected between the LPC connector and PC
+    ![nxp_connectors](../../examples/platform/k32w/doc/images/k32w-dk6-connectors.jpg)
+
+2.  Use a terminal emulator (e.g.: Putty) to connect to the UART console of the
+    accessory device. Use a baudrate of 115200.
+
+    This will grant you access to the application logs.
+
+3.  Push _Button SW2_ on the accessory device and wait 6s for factory reset of
+    the device.
+4.  Find a message similar to the following one in the application logs:
+
+        I: 666[SVR] Copy/paste the below URL in a browser to see the QR Code:
+                https://dhrishi.github.io/connectedhomeip/qrcode.html?data=CH%3AI34DV%2A-00%200C9SS0
+
+5.  Open the URL in a web browser to have the commissioning QR code generated.
+
+<hr>
+
+<a name="commissioning-accessory"></a>
+
+## Commissioning accessory device
+
+To commission the accessory device onto the Thread network created in the
+[Forming Thread network](#Forming-a-Thread-network) section, complete the
+following steps:
+
+1. Enable _Bluetooth_ and _Location_ services on your smartphone;
+2. Connect the smartphone to _OT-BR_ WiFi network;
+3. Open the CHIPTool application on your smartphone;
+4. Tap the _PROVISION CHIP DEVICE WITH THREAD_ button and scan the commissioning
+   QR code. Several notifications will appear, informing you of commissioning
+   progress with scanning, connection, and pairing. At the end of this process,
+   the Thread network settings screen appears.
+
+    ![chiptool_main_screen](../../examples/platform/k32w/doc/images/chiptool_main_screen.png)
+
+5. In the Thread network settings screen, use the default settings and tap the
```

Comment:
Again, are these default settings pre-filled or to be copied from the image?

Reply:
Pre-filled.